app/celer-sim/Runner.cc

```diff
@@ -598,15 +598,19 @@ void Runner::build_optical_collector(RunnerInput const& inp,
     }
     CELER_ASSERT(inp.optical);
 
+    size_type num_streams = core_params_->max_streams();
+
     OpticalCollector::Input oc_inp;
     oc_inp.material = MaterialParams::from_import(
         imported, *core_params_->geomaterial(), *core_params_->material());
     oc_inp.cerenkov = std::make_shared<CerenkovParams>(oc_inp.material);
     oc_inp.scintillation
         = ScintillationParams::from_import(imported, core_params_->particle());
-    oc_inp.buffer_capacity = inp.optical.buffer_capacity;
-    oc_inp.primary_capacity = inp.optical.primary_capacity;
-    oc_inp.auto_flush = inp.optical.auto_flush;
+    oc_inp.num_track_slots = ceil_div(inp.optical.num_track_slots, num_streams);
```

Comment:
I suppose we should divide the optical track/buffer sizes by the number of streams too?

Reply:
We're already doing it for the core parameters in celer-sim, so should I keep it for these optical parameters too, or no? At some point we had decided to do this in celer-sim and not celer-g4, though I can't remember the reasoning now...

Reply:
Let's keep it as consistent as possible for now? 😅 though it sounds like it might be too late...